assets/server.ts

```diff
@@ -70,12 +80,12 @@ class Server {
      * @param {Object} data
      * @return {Promise}
      */
-    post(url: any, data: any, etag?: string) {
+    post(url: any, data: any, etag?: string, rawResponse = false) {
```

Comment:
would rather put it in an options object:
```ts
post(url: any, data: any, etag?: string, {rawResponse}: {rawResponse: bool})
```
 so you see what is going on when looking at the usage:
```ts
await server.post(url, payload, undefined, {rawResponse: true});
```
and I would consider calling it `parseJson: false` or `processResponse: false`
so it's more obvious it's trying to avoid something